bh_core.sublime-settings

```diff
@@ -559,6 +559,26 @@
             "language_filter": "whitelist",
             "language_list": ["CMake"],
             "enabled": true
+        },
+        // SINUMERIK 840D SL G-Code
+        {
+            "name": "s840d_gcode",
+            "open": "(?i)\\b(IF(?!.*GOTO)|FOR|WHILE|REPEAT(?!.*GOTO))\\b",
```

Comment:
Don't use (?i), it should already be case insensitive.  If you need to enforce case, do it in a bh_module.  These flags will affect all currently active rules.